sdk/contentsafety/Azure.AI.ContentSafety/tsp-location.yaml

```diff
@@ -0,0 +1,4 @@
+# commit: 345dd133648fea2046f2fb484111ab4dba5ff34b
+# repo: https://github.com/mengaims/azure-rest-api-specs-pr/
+directory: specification/cognitiveservices/ContentSafety
+spec-root-dir: D:\src\mengaims\azure-rest-api-specs-pr
```

Comment:
We shouldn't be putting local paths that won't resolve on most machines in the repo. @m-nash is this expected?

Reply:
The fact is that it won't work on most people's machines seems to indicate it's not even necessary.

Reply:
Yeah it won't pass CI either like this.

Reply:
Removed this file.